### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by metaclips**
**Resolves**
#4

**What's new**
Fixed go modules dcrlibwallet import
Added main package
Fixed code breakage due to dcrlibwallet version

### CODE REVIEW

**Comment by oluwandabira** [comment]
Add a screenshot/gif showcasing the feature.

File: go.mod [hunk 1]
```diff
@@ -8,12 +8,9 @@ require (
...
-	github.com/raedahgroup/dcrlibwallet v1.1.0
+	github.com/raedahgroup/dcrlibwallet v1.1.1-0.20200112073453-40a403ab6f3a
 	golang.org/x/exp v0.0.0-20191002040644-a1355ae1e2c3
 	golang.org/x/image v0.0.0-20190802002840-cff245a6509b
 )
 
-// TODO: Remove and use an actual release of dcrlibwallet
-replace github.com/raedahgroup/dcrlibwallet => github.com/raedahgroup/dcrlibwallet v1.1.1-0.20200113081741-39a55988f78c
-
 replace github.com/raedahgroup/dcrlibwallet/spv => github.com/raedahgroup/dcrlibwallet/spv v0.0.0-20200113081741-39a55988f78c
```
Comment (oluwandabira) [inline go.mod:17]: Keep the `TODO` because this just looks wrong.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import (
+	"github.com/decred/dcrd/dcrutil"
+)
+
+const appDisplayName = "GoDCR"
+
+var defaultAppDataDir = dcrutil.AppDataDir("godcr", false)
+
+func main() {
+	launchUserInterface(appDisplayName, defaultAppDataDir, "testnet3")
```
Comment (oluwandabira) [inline main.go]: Does it only work with "testnet3"?

File: pages/common/syncer.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ type widgetItems struct {
 }
 
 type Syncer struct {
-	err error
+	// err error
```
Comment (oluwandabira) [inline pages/common/syncer.go]: If it is not being used, remove it.

File: pages/wallet/create.go [hunk 4]
```diff
@@ -19,8 +19,8 @@ type (
 		seedPage          *SeedPage
 		seed              string
 		isShowingSeedPage bool
-		isCreating        bool
-		err               error
+		// isCreating        bool
```
Comment (oluwandabira) [inline pages/wallet/create.go]: If it is not being used, remove it.

File: pages/wallet/create.go [hunk 5]
```diff
@@ -61,14 +61,14 @@ func (w *CreateWalletPage) cancel() {
 }
 
 func (w *CreateWalletPage) create() {
-	w.pinAndPasswordWidget.IsCreating = true
+	//	w.pinAndPasswordWidget.IsCreating = true
```
Comment (oluwandabira) [inline pages/wallet/create.go]: Why do this?

File: pages/wallet/create.go [hunk 6]
```diff
@@ -78,6 +78,6 @@ func (w *CreateWalletPage) create() {
 	}()
 
 	<-doneChan
-	w.pinAndPasswordWidget.IsCreating = false
+	//	w.pinAndPasswordWidget.IsCreating = false
```
Comment (oluwandabira) [inline pages/wallet/create.go]: Why?

File: pages/wallet/seed.go [hunk 7]
```diff
@@ -23,7 +23,7 @@ type (
 	}
 
 	seedScreen struct {
-		err                    error
+		// err                    error
```
Comment (oluwandabira) [inline pages/wallet/seed.go]: Why?

File: widgets/button.go [hunk 8]
```diff
@@ -36,10 +36,10 @@ type (
 
 	Icon struct {
 		imgSize int
-		img     image.Image
-		src     []byte
-		color   color.RGBA
-		op      paint.ImageOp
+		// img     image.Image
```
Comment (oluwandabira) [inline widgets/button.go]: Why?

File: widgets/label.go [hunk 9]
```diff
@@ -205,16 +205,16 @@ func getTextAlignment(alignment Alignment) text.Alignment {
...
+// func getLayoutAlignment(alignment Alignment) layout.Alignment {
+// 	switch alignment {
+// 	case AlignMiddle:
+// 		return layout.Middle
+// 	case AlignRight:
+// 		return layout.End
+// 	default:
+// 		return layout.Start
+// 	}
+// }
```
Comment (oluwandabira) [inline widgets/label.go]: Can you stop just commenting functions out with no reason? If it is not being used it should not be in the code base.

File: widgets/security/passwordAndPin.go [hunk 10]
```diff
@@ -23,7 +23,7 @@ type (
 	pinTab struct {
 		pinInput        *editor.Input
 		confirmPinInput *editor.Input
-		pinStrength     int
+		//	pinStrength     int
```
Comment (oluwandabira) [inline widgets/security/passwordAndPin.go]: .

File: widgets/tabcontainer.go [hunk 11]
```diff
@@ -81,11 +81,11 @@ func (t *TabContainer) drawNavSection(ctx *layout.Context) {
...
+// func (t *TabContainer) drawContentSection(ctx *layout.Context, renderFuncs []func(*layout.Context)) {
+// 	inset := layout.Inset{
+// 		Top: unit.Dp(25),
+// 	}
+// 	inset.Layout(ctx, func() {
 
-	})
-}
+// 	})
+// }
```
Comment (oluwandabira) [inline widgets/tabcontainer.go]:
```go
// Stop with the comments.
```

**Comment by metaclips** [comment]
Updated

**Review by oluwandabira — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (metaclips) [reply main.go]: we are currently testing on testnet

**Comment by oluwandabira** [comment]
Crop the screenshot.

**Review by oluwandabira — COMMENTED** [review]
The code looks better now. However I'm not so sure about the screenshot you sent. Is that how you want it to look?

File: main.go (hunk 2 above)
Comment (oluwandabira) [reply main.go]: But does the network have to be hard coded in?

**Comment by oluwandabira** [comment]
Because that doesn't look like a "Wallet Creation" page to me. Looks more like "Wallet Overview".

**Comment by oluwandabira** [comment]
Reformat the pr comment to follow the [Pull Request Template](https://github.com/raedahgroup/godcr-gio/blob/master/.github/pull_request_template.md).

**Comment by metaclips** [comment]
> Because that doesn't look like a "Wallet Creation" page to me. Looks more like "Wallet Overview".

Wallet creation page is only viewed if a wallet does not exist on the computer.

**Comment by metaclips** [comment]
> The code looks better now. However I'm not so sure about the screenshot you sent. Is that how you want it to look?

This PR only fixes build issue, no UI modification was made

**Review by oluwandabira — COMMENTED** [review]
Why is it a constant? This is effectively the same as the last code.

**Review by oluwandabira — APPROVED** [review]
Looks okay.

**Pull request merged**